deployment/templates/fluentd.yaml

```diff
@@ -31,8 +31,9 @@ objects:
     spec:
       replicas: 0
       selector:
-        provider: openshift
-        component: "fluentd"
+        matchLabels:
+          provider: openshift
+          component: "fluentd"
       triggers:
       - type: ConfigChange
       - type: ImageChange
```

Comment:
Do DaemonSets support image change triggers? Or any kind of trigger for that matter?